community/modules/compute/schedmd-slurm-gcp-v5-node-group/variables.tf

```diff
@@ -104,6 +108,24 @@ variable "instance_image" {
   }
 }
 
+variable "source_image_project" {
+  type        = string
+  description = "Alternate method to instance_image for specifying a project hosting the custom VM image."
```

Comment:
Nit: here I would explicitly say: "We recommend using 'instance_image' instead".